ios general/ios general/CreateEmployeeViewController.m

```diff
@@ -0,0 +1,60 @@
+#import "CreateEmployeeViewController.h"
+
+@interface CreateEmployeeViewController ()
+
+@end
+
+@implementation CreateEmployeeViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    self.title = @"Add new Employee";
+    
+    [self.firstNameField addTarget:self action:@selector(textFieldDidChange:) forControlEvents:UIControlEventEditingChanged];
+    [self.lastNameField  addTarget:self action:@selector(textFieldDidChange:) forControlEvents:UIControlEventEditingChanged];
+    [self.salaryField    addTarget:self action:@selector(textFieldDidChange:) forControlEvents:UIControlEventEditingChanged];
+    
+    UIBarButtonItem *saveButton = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemSave target:self action:@selector(saveNewObject:)];
+    self.navigationItem.rightBarButtonItem = saveButton;
+}
+
+- (void)didReceiveMemoryWarning
+{
+    [super didReceiveMemoryWarning];
+}
+
+-(BOOL)textFieldDidChange:(UITextField *)textField
+{
+    if ([textField.text isEqualToString:@""])
```

Comment:
@ArtemKalmykov 
`if (textField.text.length > 0)` doesn't work, but
` if (textField.text.length == 0)` does.
And can we use ` if ([textField hasText])`is that case?